service/src/main/java/bio/terra/externalcreds/dataAccess/KmsEncryptDecryptHelper.java

```diff
@@ -0,0 +1,54 @@
+package bio.terra.externalcreds.dataAccess;
+
+import bio.terra.externalcreds.ExternalCredsException;
+import bio.terra.externalcreds.config.ExternalCredsConfig;
+import bio.terra.externalcreds.config.ExternalCredsConfigInterface.KmsConfiguration;
+import com.google.cloud.kms.v1.CryptoKeyName;
+import com.google.cloud.kms.v1.DecryptResponse;
+import com.google.cloud.kms.v1.EncryptResponse;
+import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import org.springframework.stereotype.Component;
+
+/** Utils for KMS symmetric encryption and decryption. */
+@Component
+public record KmsEncryptDecryptHelper(ExternalCredsConfig config) {
+
+  /** Encrypt with KMS symmetric key. */
+  public String encryptSymmetric(String plainText) {
+    try (var client = KeyManagementServiceClient.create()) {
```

Comment:
do we actually need to create a new one of these every time or can `KeyManagementServiceClient.create()` be called just once at class instantiation and that client be used for the duration?

Reply:
the gcp documentation suggests that `After completing all of your requests, call the "close" method on the client to
safely clean up any remaining background resources.` i imagine it's not very common that we will have a sequence of frequent encrypt and decrypt except when testing.

Reply:
I looked at their examples and they are all toys. It looks to me like the client is meant for long term use. Following our use of other google client libraries, we should allocate this once.

Reply:
ok. instantiate at first method call instead of when instantiating the class as I don't want to handle api call exception in the constructor.